client/src/services/api.js

```diff
@@ -3,10 +3,10 @@ import axios from 'axios';
 const user_token = localStorage.getItem('jwt') || null ;
 // baseURL will change to heroku link
 const api = axios.create({
-    baseURL: 'https://plant-pals.herokuapp.com',
+    // baseURL: 'https://plant-pals.herokuapp.com',
+    baseURL: "http://localhost:3001",
```

Comment:
Awesome this looks great! :tada: :

Reply:
Should we keep the base url in matser plant-pals? We can probably add an environment to our local variables that automatically accounts for the change Heroku (`BASE_URL`)?

Reply:
That sounds like a good idea!